conf/switches.conf.example

```diff
@@ -6,6 +6,7 @@
 # http://www.fsf.org/licensing/licenses/gpl.html
 
 [default]
+type=Cisco::Catalyst_2960
```

Comment:
My main issue here is you are changing previous behavior with no real added benefit.
In my opinion The default switch and by extension switch group is a "power user" feature.

By making the decision for the user that default type is "Cisco::Catalyst_2960"
I believe it is easier for people to make a mistake when configuring their switches.

Also by forcing user to add the switch type for a switch by default I feel it will reduce mistakes when configuring.

If a user decides to configure the default type then they should know the danger.